lib/Listerine/commands.ex

```diff
@@ -11,6 +11,16 @@ defmodule Listerine.Commands do
   command study(roles) do
     role_list = Listerine.Helpers.upcase_words(roles)
 
+    role_list =
+      Enum.map(
+        role_list,
+        fn
+          <<year::binary-size(1), "ANO">> -> Listerine.Channels.get_roles_year(year)
```

Comment:
Can you send me the docs for this syntax?

Reply:
https://stackoverflow.com/questions/41132669/how-does-binary-size-work-in-elixir